lib/instance-actions.ts

```diff
@@ -0,0 +1,181 @@
+import cloneDeep from 'lodash/cloneDeep'
+import { AxiosInstance } from 'axios'
+import type { MetaSysProps, DefaultElements } from './common-types'
+
+import errorHandler from './error-handler'
+
+type ThisContext = { sys: MetaSysProps } & DefaultElements<{ sys: MetaSysProps }>
+
+/**
+ * @private
+ */
+export function createUpdateEntity<T = unknown>({
+  http,
+  entityPath,
+  wrapperMethod,
+  headers,
+}: {
```

Comment:
Might be worth using `@internal` and setting [`compilerOptions.stripInternal`](https://www.typescriptlang.org/v2/en/tsconfig#stripInternal) to true so that they aren't available in the generated declaration files.

Reply:
nice tip!